pkg/api/types.go

```diff
@@ -1015,6 +1015,21 @@ func (p *Properties) HasZonesForAllAgentPools() bool {
 	return false
 }
 
+// IsUbuntuDistroForAllAgentPools returns true if all of the agent pools plus masters are running the base Ubuntu image
+func (p *Properties) IsUbuntuDistroForAllAgentPools() bool {
+	if len(p.AgentPoolProfiles) > 0 {
+		for _, ap := range p.AgentPoolProfiles {
+			if ap.Distro != Ubuntu {
```

Comment:
`&& ap.Distro != Ubuntu1804` for 18.04 testing

Reply:
done